tools/generator/src/Generator+AddBazelDependenciesTarget.swift

```diff
@@ -23,7 +23,35 @@ env -i \
   "$BAZEL_PATH"
 """#
 
-    private static let setup = #"""
+    private static func setup(buildMode: BuildMode) -> String {
+        let lldbInit: String
+        if buildMode.requiresLLDBInit {
+            lldbInit = #"""
+
+if [[ -f "$HOME/.lldbinit" ]]; then
+  home_init="command source ~/.lldbinit
+
+"
+else
```

Comment:
Is this extra line for formatting in the final output below?

Reply:
Yep. Causes a new line to appear after the output, otherwise no new lines when omitted.